LRBv21.py

```diff
@@ -348,7 +353,26 @@ def LRBv21(constants,radios,d,SparRange,
         for k0 in range(si.timeout):
             
             # Initialise
-            st = iterate(si, st)
+            try:
+                st = iterate(si, st)
+            except Exception as e:
+                print(f"Iteration failed, retrying",e)
+                
+                try:
+                    st = iterate(si, st)
+                except Exception as e:
+                    print(f"Iteration failed, retrying",e)    
+                    
+                    try:
+                        st = iterate(si, st)
+                    except Exception as e:
+                        print(f"Iteration failed, retrying",e)    
+                        
+                        try:
+                            st = iterate(si, st)
+                        except Exception as e:
+                            print(f"Iteration failed, stopping")
+                            print(e)  
```

Comment:
These are here temporarily while I resolved issues with ODEINT giving spurious failures. Retrying it several times was a dirty workaround.